crates/goose-server/src/routes/schedule.rs

```diff
@@ -231,26 +231,16 @@ async fn run_now_handler(
         .await
         .map_err(|_| StatusCode::INTERNAL_SERVER_ERROR)?;
 
-    let recipe_display_name = match scheduler.list_scheduled_jobs().await {
-        Ok(jobs) => jobs
-            .into_iter()
-            .find(|job| job.id == id)
-            .and_then(|job| {
-                std::path::Path::new(&job.source)
+    let (recipe_display_name, recipe_version) = match scheduler.list_scheduled_jobs().await {
+        Ok(jobs) => {
+            if let Some(job) = jobs.into_iter().find(|job| job.id == id) {
+                let recipe_display_name = std::path::Path::new(&job.source)
                     .file_name()
                     .and_then(|name| name.to_str())
                     .map(|s| s.to_string())
-            })
-            .unwrap_or_else(|| id.clone()),
-        Err(_) => id.clone(),
-    };
+                    .unwrap_or_else(|| id.clone());
```

Comment:
should we just use the name of recipe, rather than basing something on the filename?

Reply:
I dont want to be difficult here, but the variable is called recipe_display_name, so showing the name of the recipe seems kinda reasonable. the plan is to merge recipes and scheduler either way and do away with the extra naming layers we have everywhere, fwiw